[2025-09-30T18:16:10.549+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-30T18:16:10.686+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag4.task_tasks_data manual__2025-09-30T18:16:01.029766+00:00 [queued]>
[2025-09-30T18:16:10.737+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag4.task_tasks_data manual__2025-09-30T18:16:01.029766+00:00 [queued]>
[2025-09-30T18:16:10.740+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-30T18:16:10.828+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): task_tasks_data> on 2025-09-30 18:16:01.029766+00:00
[2025-09-30T18:16:10.872+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=69) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-09-30T18:16:10.876+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag4', 'task_tasks_data', 'manual__2025-09-30T18:16:01.029766+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/dag4.py', '--cfg-path', '/tmp/tmpo4a37nic']
[2025-09-30T18:16:10.889+0000] {standard_task_runner.py:63} INFO - Started process 71 to run task
[2025-09-30T18:16:10.891+0000] {standard_task_runner.py:91} INFO - Job 8: Subtask task_tasks_data
[2025-09-30T18:16:11.030+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-09-30T18:16:11.186+0000] {task_command.py:426} INFO - Running <TaskInstance: dag4.task_tasks_data manual__2025-09-30T18:16:01.029766+00:00 [running]> on host c7a890046c7a
[2025-09-30T18:16:11.746+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag4' AIRFLOW_CTX_TASK_ID='task_tasks_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-30T18:16:01.029766+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-30T18:16:01.029766+00:00'
[2025-09-30T18:16:11.750+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-30T18:16:11.914+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-30T18:16:11.932+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-30T18:16:11.964+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag4, task_id=task_tasks_data, run_id=manual__2025-09-30T18:16:01.029766+00:00, execution_date=20250930T181601, start_date=20250930T181610, end_date=20250930T181611
[2025-09-30T18:16:12.051+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-30T18:16:12.131+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-30T18:16:12.134+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
